Add Freeze 8 warning to blood pressure GWAS

aofarrel opened the pull request:
Only changes the beginning. Adds a warning, as the crew agreed upon in the GWAS meeting this morning.

(Well, I also reformatted the changelog, but that's not important.)

Here's how it renders in Terra on the BDC skin.
<img width="1047" alt="Screenshot 2021-02-03 at 1 19 12 PM" src="https://user-images.githubusercontent.com/27784612/106811065-a64e4280-6622-11eb-808c-9acce6473542.png">

aofarrel committed:
Update BloodPressure_GWAS_Template.md

Add Freeze 8 warning.

aofarrel commented:
Originally, I was going to go with this cropped error message generated by Atom Smasher's generator website, but the generated images are either CC-A-NC or CC-A-SA, but like, it shouldn't be both, but he has both on his website, and Share-Alike might be an issue since I think we're not using CC, etc etc..

![freeze8warning-attribution-AtomSmasher](https://user-images.githubusercontent.com/27784612/106811251-eb727480-6622-11eb-9980-dbc188d08b27.png)

aofarrel committed:
Be less alarmist

bethsheets commented:
What about:
"This template workspace was developed and tested using TOPMed Freeze 5b data. Initial scale testing with Freeze 8 data has raised scalability issues related to LD pruning and generating a Genetic Relatedness Matrix within the Hail Jupyter notebook.  We do not suggest using Freeze 8 data with the Hail notebook and are looking into alternative tools. Some researchers may be able to use kinship matrices provided through the dbGAP TOPMed Combined Exchange Area. The current GENESIS WDL used for associating testing in this workspace has been updated for use with Freeze 8."

aofarrel committed:
beth's edits + some agreed upon additions from Slack

"ProTip! Great commit summaries contain fewer than 50 characters. Place extra information in the extended description."


no
